### PR TITLE
Feature/sup 43 login page

### PR DESCRIPTION
**Pull request by IrynaZazyba**
changed user authorization process, added authorization form

### CODE REVIEW

**Review by svareyko — APPROVED** [review]
Could you please share a picture of components?

File: src/main/java/by/itech/lab/supplier/auth/config/WebSecurityConfig.java [hunk 1]
```diff
@@ -33,25 +32,17 @@
...
     protected void configure(HttpSecurity http) throws Exception {
-        http
-                .csrf().disable()
-
-                .formLogin()
-                .loginPage(URL_LOGIN)
-                .permitAll()
-                .successHandler(authenticationSuccessHandler)
-                .failureHandler(authenticationFailureHandler)
+        http.cors()
```
Comment (svareyko) [inline src/main/java/by/itech/lab/supplier/auth/config/WebSecurityConfig.java]: cors?

File: frontend/src/services/requestService.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify(body)
+        };
+
+        const res = await fetch(url, requestOptions);
+        if(!res.ok){
+            throw Error(`Couldn't fetch ${url} status: ${res.status}`);
+        }
+        return await res.json();
```
Comment (svareyko) [inline frontend/src/services/requestService.js:17]: I not sure that 2nd await is necessary

Comment (IrynaZazyba) [reply frontend/src/services/requestService.js:17]: .json() return a promise

**Review by FortuneAwaker — APPROVED** [review]

**Review by alexeyoverchenko — APPROVED** [review]

**Pull request merged**